content/hi/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+
+                <p>प्रत्येक कुबेरनेट्स नोड पर काम से काम ये चलते हैं:</p>
+                <ul>
+                    <li>क्यूबलेट, कुबेरनेट्स कण्ट्रोल प्लेन और नोड के बीच संचार के लिए जिम्मेदार एक प्रक्रिया; यह पॉड्स और मशीन पर चलने वाले कंटेनरों का प्रबंधन करता है।</li>
+                    <li>एक कंटेनर रनटाइम (जैसे डॉकर), एक रजिस्ट्री से कंटेनर इमेज को पुल करने, कंटेनर को अनपैक करने और एप्लिकेशन चलाने के लिए।</li>
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i> कंटेनरों को केवल एक ही पॉड में एक साथ शेड्यूल किया जाना चाहिए यदि वे कसकर युग्मित हैं और डिस्क जैसे संसाधनों को साझा करने की आवश्यकता है। </i></p>
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2 style="color: #3771e3;">नोड अवलोकन</h2>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_03_nodes.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>kubectl के साथ समस्या निवारण</h2>
+                <p>मॉड्यूल <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a> में, आपने कमांड लाइन इंटरफ़ेस का उपयोग किया है। डेप्लॉयड एप्लीकेशन और उनके एनवायरनमेंट के बारे में जानकारी प्राप्त करने के लिए आप मॉड्यूल 3 में इसका उपयोग करना जारी रखेंगे। सबसे आम ऑपरेशन निम्नलिखित kubectl कमांड के साथ किया जा सकता है:</p>
+                <ul>
+                    <li><b>kubectl get</b> - संसाधनों की सूची बनाएं</li>
+                    <li><b>kubectl describe</b> - संसाधन के बारे में विस्तृत जानकारी दिखाएं</li>
+                    <li><b>kubectl logs</b> - पॉड में कंटेनर के लॉग प्रिंट करें</li>
+                    <li><b>kubectl exec</b> - पॉड में कंटेनर पर कमांड चलाए</li>
+                </ul>
+
+                <p>आप इन कमांड का उपयोग यह देखने के लिए कर सकते हैं कि एप्लिकेशन कब डिप्लॉय किए गए थे, उनकी वर्तमान स्थिति क्या है, वे कहां चल रहे हैं और उनके कॉन्फ़िगरेशन क्या हैं।</p>
+
+                <p>अब जब हम अपने क्लस्टर कॉम्पोनेन्ट और कमांड लाइन के बारे में जानते हैं, तो आइए हमारे एप्लिकेशन को देखें।</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i> कुबेरनेट्स में एक नोड एक कार्यकर्ता मशीन है और क्लस्टर के आधार पर एक वीएम या भौतिक मशीन हो सकती है। एक नोड पर कई पॉड चल सकते हैं। </i></p>
```

Comment:
`भौतिक` means `materialistic` in english, I think `आभासी` is the exact translation of `virtual`, what do you think?

Reply:
I think we should do `वर्चुअल`